### PR TITLE
Add installation configure of LH GuaranteedInstanceManagerCPU

### PR DESCRIPTION
**Pull request by w13915984028**
Per https://github.com/harvester/harvester/issues/5568

With backend PR https://github.com/harvester/harvester-installer/pull/706, we also need to update the document.

### CODE REVIEW

File: versioned_docs/version-v1.2/install/harvester-configuration.md [hunk 1]
```diff
@@ -742,6 +742,34 @@ install:
         guaranteedReplicaManagerCPU: 6
 ```
 
+### `install.harvester.longhorn.default_settings.guaranteedInstanceManagerCPU`
+
+_Available as of v1.2.2_
+
+#### Definition
+
+Sets the default percentage of the total allocatable CPU on each node will be reserved for each Longhorn instance manager Pod.
```
Comment (jillian-maroket) [inline versioned_docs/version-v1.2/install/harvester-configuration.md:751]:
```suggestion
Percentage of the total allocatable CPU on each node to be reserved for each Longhorn Instance Manager pod.
```

File: versioned_docs/version-v1.2/install/harvester-configuration.md [hunk 2]
```diff
@@ -742,6 +742,34 @@ install:
...
+
+Default: 12
+
+Supported values: 0-12. All other values are considered 12.
+
+This integer value indicates what percentage of the total allocatable CPU on each node will be reserved for each instance manager Pod.
+
+In edge scenarios where users may deploy single-node Harvester clusters, they can set this parameter to a value smaller than 12. In most scenarios, it is recommended to keep the default value for system high availability.
+
+Before setting the value, please refer to [longhorn-guaranteed-instance-manager-cpu](https://longhorn.io/docs/1.5.5/references/settings/#guaranteed-instance-manager-cpu) for more details.
```
Comment (jillian-maroket) [inline versioned_docs/version-v1.2/install/harvester-configuration.md:761]:
```suggestion
This integer value indicates the percentage of the total allocatable CPU on each node to be reserved for each Longhorn Instance Manager pod.

Using the default value is recommended for high system availability. When deploying single-node Harvester clusters, you can specify a value less than 12.

For more information about how to set the correct value, see [Guaranteed Instance Manager CPU](https://longhorn.io/docs/1.6.0/references/settings/#guaranteed-instance-manager-cpu) in the Longhorn documentation.
```

File: versioned_docs/version-v1.3/install/harvester-configuration.md [hunk 3]
```diff
@@ -713,51 +713,23 @@ install:
...
-### `install.harvester.longhorn.default_settings.guaranteedEngineManagerCPU`
+### `install.harvester.longhorn.default_settings.guaranteedInstanceManagerCPU`
 
-_Available as of v1.2.0_
+_Available as of v1.3.1_
 
 #### Definition
 
-Sets the default percentage of the total allocatable CPU on each node will be reserved for each Longhorn engine manager Pod.
+Sets the default percentage of the total allocatable CPU on each node will be reserved for each Longhorn instance manager Pod.
```
Comment (jillian-maroket) [inline versioned_docs/version-v1.3/install/harvester-configuration.md:722]:
```suggestion
Percentage of the total allocatable CPU on each node to be reserved for each Longhorn Instance Manager pod.
```

File: versioned_docs/version-v1.3/install/harvester-configuration.md [hunk 4]
```diff
@@ -713,51 +713,23 @@ install:
...
 
 Default: 12
 
 Supported values: 0-12. All other values are considered 12.
 
-This integer value indicates what percentage of the total allocatable CPU on each node will be reserved for each engine manager Pod.
+This integer value indicates what percentage of the total allocatable CPU on each node will be reserved for each instance manager Pod.
```
Comment (jillian-maroket) [inline versioned_docs/version-v1.3/install/harvester-configuration.md:728]:
```suggestion
This integer value indicates the percentage of the total allocatable CPU on each node to be reserved for each Longhorn Instance Manager pod.
```

File: versioned_docs/version-v1.3/install/harvester-configuration.md [hunk 5]
```diff
@@ -713,51 +713,23 @@ install:
...
-Default: 12
-
-Supported values: 0-12. All other values are considered 12.
-
-This integer value indicates what percentage of the total allocatable CPU on each node will be reserved for each replica manager Pod.
-
-In edge scenarios where users may deploy single-node Harvester clusters, can set this parameter to a value smaller than 12. In most scenarios, it is recommended to keep the default value for system high availability.
-
-Before setting the value, please refer to [longhorn-guaranteed-replica-manager-cpu](https://longhorn.io/docs/1.4.1/references/settings/#guaranteed-replica-manager-cpu) for more details.
+Before setting the value, please refer to [longhorn-guaranteed-instance-manager-cpu](https://longhorn.io/docs/1.5.5/references/settings/#guaranteed-instance-manager-cpu) for more details.
```
Comment (jillian-maroket) [inline versioned_docs/version-v1.3/install/harvester-configuration.md:732]:
```suggestion
For more information about how to set the correct value, see [Guaranteed Instance Manager CPU](https://longhorn.io/docs/1.6.0/references/settings/#guaranteed-instance-manager-cpu) in the Longhorn documentation.
```

**Review by jillian-maroket — COMMENTED** [review]
Review done

File: versioned_docs/version-v1.2/install/harvester-configuration.md (hunk 2 above)
Comment (w13915984028) [reply versioned_docs/version-v1.2/install/harvester-configuration.md]: Others are updated, but the longhorn link keeps v1.5.5 as in Harvester v1.2.2, the used LH version is v1.5.5.

**Review by jillian-maroket — APPROVED** [review]

**Review by bk201 — APPROVED** [review]
LGTM, thanks!

**Pull request merged**